src/firebase_functions/private/util.py

```diff
@@ -402,17 +402,33 @@ def get_precision_timestamp(time: str) -> PrecisionTimestamp:
         return PrecisionTimestamp.MICROSECONDS
 
 
-def timestamp_conversion(time: str) -> _dt.datetime:
-    """Converts a timestamp and returns a datetime object of the current time in UTC"""
-    precision_timestamp = get_precision_timestamp(time)
+def timestamp_conversion(time) -> _dt.datetime:
+    """
+    Converts a timestamp and returns a datetime object of the current time in UTC.
```

Comment:
this returns the current time in UTC? i know this docstring was existing, but i find it confusing?